components/aws/sagemaker/codebuild/integration-test.buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2      
+phases:
+  build:
+    commands:
+      - cd components/aws
+      - docker build . -f ./sagemaker/tests/integration_tests/Dockerfile -t amazon/integration-test-image --quiet
```

Comment:
The folder structure is different 
the structure you have used here is how it should be in integration tests PR
(you don't need to change anything)

"./sagemaker/**test/integration**/Dockerfile" - PR for integration tests : https://github.com/kubeflow/pipelines/pull/3654

Reply:
Okay yes, we should converge on the same naming scheme for all of these PRs. I think `/tests/integration_tests` would be best, considering we already have `/tests/unit_tests`.

Reply:
Will do